frontend/src/components/staff/Update.jsx

```diff
@@ -252,7 +259,10 @@ export default function Update({ refresh, onRefresh }) {
         <div>
           <div>
             {selectedItems.map((item) => (
-              <Input key={item.itemName} label={item.itemName}>
+              <Input
+                key={item.itemName + Math.floor(Math.random() * 100)}
```

Comment:
```suggestion
                key={item.itemName}
```

Reply:
i'm curious about this one, was there an overlap in itemNames?

Reply:
If the user adds the same item twice.

Reply:
But, If we assume user gives each item unique then we can keep only item name as key

Reply:
sounds good, thank you for adding that as a safety. I'll open an issue to confirm that inserting functions as we'd expect.

for future use, random would possibly create an overlap (although unlikely, it's still possible). to generate a "random" unique key, i'd suggest something like [uuid](https://www.npmjs.com/package/uuid)